lib/oidcstrategy.js

```diff
@@ -144,20 +146,26 @@ function onProfileLoaded(strategy, args) {
  *   - `responseMode`       For login only flows we should have token passed back to us in a POST
  *   - `validateIssuer`     if you have validation on, you cannot have users from multiple tenants sign in
  *   - `passReqToCallback`  if you want the Req to go back to the calling function for other processing use this.
- *
+ *   - `stateMaxAmount`     the max amount of state you want to save in session, default is 200
+ *   - `stateMaxAge`        the lifetime of state in session, default is 1800s = 30 minutes
+ *   - `nonceMaxAmount`     the max amount of nonce you want to save in session, default is 200
+ *   - `nonce`              the lifetime of nonce in session, default is 1800s = 30 minutes
  * Examples:
  *
  * passport.use(new OIDCStrategy({
  *   callbackURL: config.creds.returnURL,
- *   realm: config.creds.realm,
  *   scope: config.creds.scopes,
  *   clientID: config.creds.clientID,
  *   identityMetadata: config.creds.identityMetadata,
  *   responseType: config.creds.responseType,
  *   responseMode: config.creds.responseMode,
  *   validateIssuer: config.creds.validateIssuer,
  *   passReqToCallback: config.creds.passReqToCallback,
- *   loggingLevel: config.creds.loggingLevel
+ *   loggingLevel: config.creds.loggingLevel,
+ *   stateMaxAmount: config.creds.stateMaxAmount,
```

Comment:
We use maxAmount because here we are storing things in session in server, we need to set a limit so the array won't get too large. Wilson doesn't need maxAmount because it stores things in the cookie. 

In Wilson, state string is part of the nonce string, so it makes sense to say NonceLifetime. Here the lifetime is for state, nonce, policy (for B2C), and they are 3 separated strings.